tests/src/e2e/setup.rs

```diff
@@ -441,18 +441,62 @@ pub fn working_dir() -> PathBuf {
 
 /// A command under test
 pub struct AnomaCmd {
-    pub session: PtySession,
+    pub session: Session,
     pub cmd_str: String,
 }
 
+/// A command under test running on a background thread
+pub struct AnomaBgCmd {
+    join_handle: std::thread::JoinHandle<AnomaCmd>,
+    abort_send: std::sync::mpsc::Sender<()>,
+}
+
+impl AnomaBgCmd {
+    /// Re-gain control of a background command (created with
+    /// [`AnomaCmd::bg()`]) to check its output.
+    pub fn fg(self) -> AnomaCmd {
```

Comment:
I know it means more typing down the road, but can we rename this to `foreground` and rename bg to `background`? I think these commands will look mysterious otherwise.

Reply:
good point, thanks! I renamed them